### ISSUE TITLE
Decoder Flags

### ISSUE THREAD

**Issue by seank** [malcom2073/emstune#9]
Since the new decoder arch, the doc widget on the right(red/green fields), doesn't change with the flags. The field name "Decoder Flags" will never change in the future. The complete descriptor is here:
			.start = offsetof(DecoderStats, decoderFlags),
			.size = bitSize(DecoderStats, decoderFlags),
			.isSigned = 0,
			.name = "Decoder Flags",
			.description = "Flags representing various decoder states",
			.multiplier = "1",
			.adder = "0",
			.transfer_function = "x",
			.flags = "",
			.suffix = "",
                        .CANid="200",
			.bitFieldDescription = {
			  {.fieldName = "PreviousSecondaryEventValid", .bitPosition = 5},
			  {.fieldName = "PrimaryPeriodValid",          .bitPosition = 4},
			  {.fieldName = "PreviousPrimaryEventValid",   .bitPosition = 3},
			  {.fieldName = "CamLock",                     .bitPosition = 2},
			  {.fieldName = "CrankLock",                   .bitPosition = 1},
			  {.fieldName = "MinimalLock",                 .bitPosition = 0}
			},

Thanks!

**Comment by malcom2073** [comment]
Good call. I'm ignoring PreivousSecondaryEventValid for the time being, I'll keep this issue open until I fix that, and make the flags more dynamic/

**Comment by seank** [comment]
Sounds good.
